README.md

```diff
@@ -439,10 +439,9 @@ AWSXRayRecorder.Instance.AddAnnotation("mykey", "my value");
 AWSXRayRecorder.Instance.AddMetadata("my key", "my value");
 ```
 
-### AWS Lambda support (.NET Core)
-
-You can create `Subsegment` inside lambda function.     
-*Note*: The AWS Lambda execution environment creates a `Segment` before the Lambda function is invoked, so a `Segment` cannot be created inside the Lambda function.
+### AWS Lambda support (.NET Core) 
+You can only create and close `Subsegment` inside a lambda function.     
```

Comment:
This should be more explicit in the sense, custom subsegments, and the rest middlewares which generates subsegment such as HTTP client handler, sql and AWS SDK work inside Lambda function.